ansible/roles/nginx-proxy/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+persistant_data_path: "/mnt/persistant"
+
```

Comment:
this file will potentially get bigger. I propose the following
```
nextcloud:
   enabled: false
   nextcloud_internal_ip: 10.10.10.3
 

```

Reply:
Good idea, done